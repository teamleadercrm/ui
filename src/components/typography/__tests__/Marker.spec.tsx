import React from 'react';
import { render } from '@testing-library/react';
import { Marker } from '../';

describe('Component - Marker', () => {
  it('renders', async () => {
    const { asFragment } = render(<Marker>This is a Tag</Marker>);
    expect(asFragment()).toMatchSnapshot();
  });
});
