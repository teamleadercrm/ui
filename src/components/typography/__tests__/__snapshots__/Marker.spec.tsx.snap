// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Marker renders 1`] = `
<DocumentFragment>
  <mark
    class="box margin-left-negative-1 margin-right-negative-1 padding-left-1 padding-right-1 marker"
    data-teamleader-ui="marker"
  >
    This is a Tag
  </mark>
</DocumentFragment>
`;
